Tests/RockLib.Configuration.ObjectFactory.Tests/ConstructorOrderInfoTests.cs

```diff
@@ -11,6 +11,13 @@ namespace Tests
 {
     public class ConstructorOrderInfoTests
     {
+        private static Type _constructorOrderInfoType;
+
+        static ConstructorOrderInfoTests()
+        {
+            _constructorOrderInfoType = Type.GetType("RockLib.Configuration.ObjectFactory.ConstructorOrderInfo, RockLib.Configuration.ObjectFactory");
```

Comment:
This looks like fun.  Maybe it would be better in a shared place in case the type name changes like Rocket.Configuraiton...